pyramid.cpp

```diff
@@ -0,0 +1,18 @@
+#include<stdio.h>
+int main()
+{
```

Comment:
please put this `{` on the same line as main()
The line should look like this ```int main() {```

Standard way of writing a program
```
function myFunction(x, y) {
  if (y === undefined) {
    y = 0;
  }
}
```

Reply:
hello

Reply:
`{` is called curly bracket